pages/windows/logoff.md

```diff
@@ -0,0 +1,19 @@
+# logoff
+
+> Terminate a login session.
+
+- Terminate the current session:
+
+`logoff`
+
+- Terminate a session by its name or id:
+
+`logoff {{session_name|session_id}}`
+
+- Terminate a session on a specific RDP server:
+
+`logoff /server:{{servername}}`
+
+- Terminate a session within a server or virtual machine (session id is required):
```

Comment:
If you say "server or virtual machine", then should it be /{{server|vm}} ?

Reply:
`/vm` is the actual flag.

Reply:
alright, so should the description say server also ? Or rather then the question is what is the /server flag for ? Only remote hosts ?